avm/res/insights/action-group/tests/e2e/max/main.test.bicep

```diff
@@ -78,5 +80,15 @@ module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem'
       Environment: 'Non-Prod'
       Role: 'DeploymentValidation'
     }
+
```

Comment:
I'd suggest to remove the workaround for anything which is neither defaults nor waf-aligned.

Reply:
Removed @eriqua

Reply:
done @eriqua